examples/md-flexible/tests/CMakeLists.txt

```diff
@@ -31,14 +49,6 @@ target_include_directories(
     PUBLIC ${PROJECT_SOURCE_DIR}/tests/testAutopas ${PROJECT_SOURCE_DIR}/examples/md-flexible
 )
 
-target_link_libraries(
-    mdFlexTests
-    PUBLIC
-        autopas
-        autopasTools
-        gmock # gmock includes the gtest target
-        yaml-cpp
-)
```

Comment:
also reinstate this and link against mpi with a generator expression as shown in the suggestion in the other cmake file